node/src/Context.cpp

```diff
@@ -30,18 +30,33 @@ namespace node {
             if (optionsObject.Has("powerPreference")) {
                 std::string powerPreferenceOption = optionsObject.Get("powerPreference").ToString();
```

Comment:
Should I check if the content of the `powerPreference` is a String before I convert it ?

Reply:
Yes, please add the condition.

Reply:
I have added the condition. PTAL.